### DISCUSSION TITLE
Table Hover to show more info ?

### DISCUSSION THREAD

**Discussion by Winniepoom** [Budibase/budibase#9904] [Help] [answered]
Hello I wonder if Budibase can do something like showing more info when your hovering on Table and show data of user Something like this.
Thankyou.
![image](https://user-images.githubusercontent.com/107398226/223333679-8cce4e66-f8fd-44de-850a-82479adfae7d.png)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Winniepoom,

Currently this isn't possible with the tables component, however I will mark this as a possible enhancement. 

If you really want this feature you could try and build a custom component for it? You could pull the current table code and work from there.
